config/mail.php

```diff
@@ -257,23 +143,5 @@
         'name' => env('MAIL_REPLYTO_NAME', null),
     ],
 
-    /*
-    |--------------------------------------------------------------------------
-    | Markdown Mail Settings
-    |--------------------------------------------------------------------------
-    |
-    | If you are using Markdown based email rendering, you may configure your
-    | theme and component paths here, allowing you to customize the design
-    | of the emails. Or, you may simply stick with the Laravel defaults!
-    |
-    */
-
-    'markdown' => [
-        'theme' => 'default',
-
-        'paths' => [
-            resource_path('views/vendor/mail'),
```

Comment:
This path is still populated - is removing this needed? Or is it a default that's now fully-baked in, and thus no longer needed in this config file?